2-mandatory/1-create-functions.js

```diff
@@ -44,7 +60,21 @@ Write a function that:
 - Numbers greater 100 must be replaced with 100.
 */
 
-function formatPercentage() {
+function formatPercentage(arr) {
+  let percentArr = arr.slice();
+  return percentArr.map(numChecker);
+}
+
+function numChecker(num) {
+  if (num > 100) {
+    return "100%";
+  } else if (Number.isInteger(num)) {
+    return num + "%";
+  } else if (num < 1) {
+    return num.toFixed(2) + "%";
+  } else {
+    return num.toFixed(1) + "%";
```

Comment:
I remember, this was to satisfy the tests right. Reminder for me to chase this up with the syllabus team